app/src/debugTools/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugModule.kt

```diff
@@ -13,22 +13,29 @@ import javax.inject.Singleton
 class DebugModule @Inject constructor() {
     private val cryingProbabilityEvents: PublishSubject<Float> = PublishSubject.create()
     private val notificationEvents: PublishSubject<String> = PublishSubject.create()
+    private val soundEvents: PublishSubject<Int> = PublishSubject.create()
 
     fun sendCryingProbabilityEvent(cryingProbability: Float) {
         cryingProbabilityEvents.onNext(cryingProbability)
     }
 
     fun sendNotificationEvent(notificationInformation: String) {
-        val currentDateTimeString = LocalDateTime.now().format(DateTimeFormatter.ofPattern(DATE_TIME_PATTERN))
+        val currentDateTimeString =
+            LocalDateTime.now().format(DateTimeFormatter.ofPattern(DATE_TIME_PATTERN))
         notificationEvents.onNext("$notificationInformation at $currentDateTimeString")
     }
 
+    fun sendSoundEvent(decibels: Int) {
+        soundEvents.onNext(decibels)
+    }
+
     fun debugStateObservable(): Observable<DebugState> {
-        return Observable.combineLatest<String, Float, DebugState>(
+        return Observable.combineLatest<String, Float, Int, DebugState>(
             notificationEvents.startWith(NOTIFICATION_INFORMATION_INITIAL_STATE),
             cryingProbabilityEvents.startWith(CRYING_PROBABILITY_INITIAL_STATE),
-            BiFunction<String, Float, DebugState> { notificationInformation: String, cryingProbability: Float ->
-                DebugState(notificationInformation, cryingProbability)
+            soundEvents.startWith(SOUND_INITIAL_STATE),
+            Function3<String, Float, Int, DebugState> { notificationInformation: String, cryingProbability: Float, decibels: Int ->
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/89ff9ffb603a83b79e177dd00519720c6a8d52ac/app/src/debugTools/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugModule.kt#L37"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>